### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by tmicomyiza**
master will contain version 1.

### CODE REVIEW

**Comment by mayahp** [comment]
@googlebot I consent.

File: frontend/src/components/SignInUpModal.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+      error: null,
+      initialTab: 'login',
+      recoverPasswordSuccess: null,
+      credentials: null,
+    };
+  }
+
+  updateRedux(userCredentials, userLoggedIn) {
+    console.log('updating redux')
+    console.log(this.reduxState)
```
Comment (Jkim-Hack) [inline frontend/src/components/SignInUpModal.js:42]: Remove console logs here, api keys can be shown

File: frontend/src/components/Profile.js [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+const mapStateToProps = state => {
+  return { articles: state.articles };
+}
+
+class Profile extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.reduxState = this.props.articles[0];
+    console.log(this.reduxState);
```
Comment (Jkim-Hack) [inline frontend/src/components/Profile.js:33]: Remove console log here, api keys can be released

File: frontend/src/components/SignInUpModal.js (hunk 1 above)
Comment (tmicomyiza) [reply frontend/src/components/SignInUpModal.js:42]: I'm going to remove all the console.logs

**Pull request merged**